lib/fastlyPromise.js

```diff
@@ -92,8 +93,11 @@ FastlyPromise.prototype.request = function request(method, url, options) {
   //Set/Overwrite requestPromiseOptions
   if (options.requestPromiseOptions) { requestPromiseOptions = ramda.merge(requestPromiseOptions, options.requestPromiseOptions); }
 
-  //Fire request
-  return requestP(requestPromiseOptions);
+  //Fire request, process response.
+  return requestP(requestPromiseOptions)
+    .then(function(response){
+      return (response.headers['content-type'] == 'application/json') ? JSON.parse(response.body) : response.body;
```

Comment:
I believe this is an API-breaking change which means it should be 2.0.0 (in package.json)